### PR TITLE
Hyperledger Indy 2023 Q1 Report

### PR DESCRIPTION
**Pull request by swcurran**
Signed-off-by: Stephen Curran <swcurran@gmail.com>

### CODE REVIEW

File: project-reports/2023/2023-Q1-Hyperledger-Indy.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+increased recently, with more and more interested parties showing up,
+and new contributors weighing in on the conversations.
+
+## Contributor Diversity
+
+The edX course about Indy, Aries, AnonCreds, and Ursa ([LFS172x - Introduction
+to Hyperledger Self Sovereign Identity Blockchain Solutions]) was updated
+recently, with the new version launched in January 2023.
+
+[LFS172x - Introduction to Hyperledger Self Sovereign Identity Blockchain Solutions]: https://www.edx.org/course/identity-in-hyperledger-aries-indy-and-ursa
```
Comment (tkuhrt) [inline project-reports/2023/2023-Q1-Hyperledger-Indy.md]: What does this have to do with Contributor Diversity? Is this intended for the Additional Information section?

File: project-reports/2023/2023-Q1-Hyperledger-Indy.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+A focus on pushing the "did:indy" DID Method is enabled by the upgrade of Indy
+Node to the new Ubuntu 20.04-based releases.
+
+## Maintainer Diversity
+
+The bi-weekly Indy Contributors call continues to be the medium by which
+maintainers coordinate work, discuss critical issues to the Indy
+codebase, and agree on larger changes. Topics and attendance has
+increased recently, with more and more interested parties showing up,
+and new contributors weighing in on the conversations.
```
Comment (tkuhrt) [inline project-reports/2023/2023-Q1-Hyperledger-Indy.md]: For this section, the TOC is interested in understanding what organizations the maintainers are from. Do they all fall under a single org? Is there diversity in the maintainers?

File: project-reports/2023/2023-Q1-Hyperledger-Indy.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+[did:indy Specification]: https://hyperledger.github.io/indy-did-method/
+[did:indy Specification source repository]: https://github.com/hyperledger/indy-did-method
+[did:indy Networks]: https://github.com/hyperledger/indy-did-networks
+[Indy HIPE]: https://github.com/hyperledger/indy-hipe
+
+## Project Health
+
+A major milestone was hit in the Hyperledger Indy project this quarter.  The
+Indy Node Continuous Integration/Continuous Delivery (CI/CD) pipeline to
+automated the build, testing and publishing of the Indy ledger artifacts was
```
Comment (tkuhrt) [inline project-reports/2023/2023-Q1-Hyperledger-Indy.md]:
```suggestion
automate the build, testing and publishing of the Indy ledger artifacts was
```

File: project-reports/2023/2023-Q1-Hyperledger-Indy.md (hunk 1 above)
Comment (swcurran) [reply project-reports/2023/2023-Q1-Hyperledger-Indy.md]: When I took over doing the Indy/Aries reports, that was the type of information in the section -- activities that led to Contributor Diversity.

I've moved it to the `Additional Information section` as requested and added a link to the activity dashboard in the contributor diversity. As I review the other Quarterly reports I'll look for other information to add next report.

File: project-reports/2023/2023-Q1-Hyperledger-Indy.md (hunk 2 above)
Comment (swcurran) [reply project-reports/2023/2023-Q1-Hyperledger-Indy.md]: Same as with previous comment -- a leftover from previous reports. I'll see what additional information can be added next quarter. Maintainers are listed in each repo, but getting a consolidated list across repos will take a script of some kind.

Perhaps each project should have a maintainer team hierarchy -- a "project-name" maintainers team with all maintainers across all repos, and a per repo maintainers team with members all a part of the project team.

File: project-reports/2023/2023-Q1-Hyperledger-Indy.md (hunk 3 above)
Comment (swcurran) [reply project-reports/2023/2023-Q1-Hyperledger-Indy.md]: Fixed.

File: project-reports/2023/2023-Q1-Hyperledger-Indy.md [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+A focus on pushing the "did:indy" DID Method is enabled by the upgrade of Indy
+Node to the new Ubuntu 20.04-based releases.
+
+## Maintainer Diversity
+
+No change the Maintainers this quarter.
+
+## Contributor Diversity
+
+See the [Indy Activity Quarterly Dashboard] for information about contributors
```
Comment (tkuhrt) [inline project-reports/2023/2023-Q1-Hyperledger-Indy.md]: Missing link

File: project-reports/2023/2023-Q1-Hyperledger-Indy.md [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+## Additional Information
+
+- Key channels on Hyperledger Discord: \#indy, \#indy-sdk,
+\#indy-node, \#indy-maintainers, \#indy-vdr
+- [Indy Mailing List](https://lists.hyperledger.org/g/indy)
+- The edX course about Indy, Aries, AnonCreds, and Ursa ([LFS172x - Introduction
+to Hyperledger Self Sovereign Identity Blockchain Solutions]) was updated
+recently, with the new version launched in January 2023.
+
+   - [LFS172x - Introduction to Hyperledger Self Sovereign Identity Blockchain Solutions]: https://www.edx.org/course/identity-in-hyperledger-aries-indy-and-ursa
```
Comment (tkuhrt) [inline project-reports/2023/2023-Q1-Hyperledger-Indy.md]:
```suggestion
[LFS172x - Introduction to Hyperledger Self Sovereign Identity Blockchain Solutions]: https://www.edx.org/course/identity-in-hyperledger-aries-indy-and-ursa
```

Comment (swcurran) [reply project-reports/2023/2023-Q1-Hyperledger-Indy.md]: Fixed

File: project-reports/2023/2023-Q1-Hyperledger-Indy.md (hunk 4 above)
Comment (swcurran) [reply project-reports/2023/2023-Q1-Hyperledger-Indy.md]: Fixed

**Review by tkuhrt — APPROVED** [review]

**Review by TimoGlastra — APPROVED** [review]

**Review by lehors — APPROVED** [review]

**Review by bobbi28 — APPROVED** [review]

**Review by VRamakrishna — APPROVED** [review]

**Review by denyeart — APPROVED** [review]

**Review by mbrandenburger — APPROVED** [review]

File: project-reports/2023/2023-Q1-Hyperledger-Indy.md (hunk 2 above)
Comment (petermetz) [reply project-reports/2023/2023-Q1-Hyperledger-Indy.md]: > Perhaps each project should have a maintainer team hierarchy -- a "project-name" maintainers team with all maintainers across all repos, and a per repo maintainers team with members all a part of the project team.

@swcurran Sounds like a good idea to me. The GitHub REST API can be used to traverse organization repos so a script that uses that could be nice.

**Review by petermetz — APPROVED** [review]
@swcurran LGTM, thank you! 
It was great to read about the work about supporting Ubuntu 20. That reminded me that I wanted to put it on your radar that Ubuntu 22 is now also in LTS, so if there's willingness among the project maintainers/contributors that could also be a nice target to aim for in terms of support/test coverage in the near/mid/far future (this is all just advisory information - you do what you gotta do)

File: project-reports/2023/2023-Q1-Hyperledger-Indy.md (hunk 2 above)
Comment (ryjones) [reply project-reports/2023/2023-Q1-Hyperledger-Indy.md]: > Same as with previous comment -- a leftover from previous reports. I'll see what additional information can be added next quarter. Maintainers are listed in each repo, but getting a consolidated list across repos will take a script of some kind.
> 
> Perhaps each project should have a maintainer team hierarchy -- a "project-name" maintainers team with all maintainers across all repos, and a per repo maintainers team with members all a part of the project team.

@swcurran talk to me on discord
